sympy/printing/tests/test_latex.py

```diff
@@ -1434,3 +1434,15 @@ def test_issue_7117():
 
 def test_issue_2934():
     assert latex(Symbol(r'\frac{a_1}{b_1}')) == '\\frac{a_1}{b_1}'
+
+def test_Vector():
+    from sympy.vector.coordsysrect import CoordSysCartesian
+    A = CoordSysCartesian('A')
+    i, j, k = A.base_vectors()
+    a, b, c, d, omega = symbols('a, b, c, d, omega')
+    v = (a ** 2 + b / c) * A.i + sqrt(d) * A.j + cos(omega) * A.k
+    assert v._latex() == (r'(a^{2} + \frac{b}{c})\mathbf{\hat{a}_i} + '
+                          r'\sqrt{d}\mathbf{\hat{a}_j} + '
+                          r'\operatorname{cos}\left(\omega\right)'
+                          r'\mathbf{\hat{a}_k}')
+
```

Comment:
On current master,
```python
>>> A = CoordSys3D('A')
>>> i, j, k = A.base_vectors()
>>> a, b, c, d, omega = symbols('a, b, c, d, omega')
>>> v = (a ** 2 + b / c) * A.i + sqrt(d) * A.j + cos(omega) * A.k
>>> v._latex()
Traceback (most recent call last):
  File "/usr/lib/python3.6/code.py", line 91, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
AttributeError: 'VectorAdd' object has no attribute '_latex'
```

Reply:
`CoordSysCartesian` is now deprecated, so I'm not sure this is an interesting test.

Also, we should not be testing the `_latex` methods directly, but instead testing either `latex(v)` or `vlatex(v)`

Reply:
On current master:
```
(a^{2} + \frac{b}{c})\mathbf{\hat{i}_{A}} + (\sqrt{d})\mathbf{\hat{j}_{A}} + (\cos{\left(\omega \right)})\mathbf{\hat{k}_{A}}
```

Which differs only in `\hat{i}_{A}` vs `\hat{a}_i